### DISCUSSION TITLE
Amount of devices per emulator and multiple bridges

### DISCUSSION THREAD

**Discussion by mark-playford** [diyhue/diyHue#986] [General]
Hi guys,

We have the issue with the hue where we need more devices than the bridge can handle. Does this handle more?

We also would love 2 bridges per account, but I'm not sure if this would work.

Many Thanks
Mark

**Comment by mariusmotea** [comment]
We have no limit on diyhue for the number of devices, however diyhue don't work with Hue accounts since is not a real Hue Bridge.

**Comment by mark-playford** [comment]
Thanks very much.

Does it have a gui accessible via ip?

Many Thanks
Mark

**Reply by mariusmotea** [reply]
Yes, you can change light states from build in GUI + some additional settings.
